src/isomorphic/classic/types/checkReactTypeSpec.js

```diff
@@ -17,6 +17,17 @@ var ReactPropTypesSecret = require('ReactPropTypesSecret');
 var invariant = require('invariant');
 var warning = require('warning');
 
+var ReactComponentTreeDevtool;
+
+if (process.env.NODE_ENV === 'test') {
```

Comment:
Aah, `__DEV__` “works” because it’s how it’s called internally! :steve_jobs_fireworks: